crates/goose-mcp/src/developer/mod.rs

```diff
@@ -70,9 +70,9 @@ pub fn load_prompt_files() -> HashMap<String, Prompt> {
                 description: arg.description,
                 required: arg.required,
             })
-            .collect();
+            .collect::<Vec<PromptArgument>>();
```

Comment:
unrelated to this PR: i don't think we need this mapping  - https://github.com/block/goose/blob/main/crates/goose-mcp/src/developer/prompts/unit_test.json

it'd be easier to stay closer to the MCP prompt in the json to avoid this mapping

Reply:
i'd call it `description` instead of template for example and stay closer to the hierarchy in MCP spec

Reply:
nvm maybe we do need the template cause i don't see any way to replace the arguments - https://spec.modelcontextprotocol.io/specification/2024-11-05/server/prompts/#text-content